Cargo.toml

```diff
@@ -15,13 +15,13 @@ repository = "https://github.com/dusk-network/poseidon252"
 dusk-bls12_381 = {version = "0.6", default-features = false}
 dusk-jubjub = {version = "0.8", default-features = false}
 dusk-bytes = "0.1"
-dusk-hades = { version = "0.14", default-features = false }
+dusk-hades = { git="https://github.com/dusk-network/hades252", branch="vlopes11/update-plonk", default-features = false }
```

Comment:
Do a pre-release for Hades and use it here if it's really required and you need to merge that to release also Poseidon. 

You can check the plonk repo to see the exact workflow

Reply:
Fixed, hades `0.15.0-pre.0` published